modules/indexing/src/test/java/org/apache/ignite/internal/mxbean/SqlQueryMXBeanImplSqlFreeMemTest.java

```diff
@@ -95,4 +95,32 @@ private void validateMemoryUsageOn(int nodeIdx, MemValidator validator) throws E
 
         validator.validate(free, maxMem);
     }
+
```

Comment:
In order to not overcomplicate logic by supporting this test(SqlQueryMXBean is deprecated), I decided to copy here some validations